Xamarin.Forms.Core/Picker.cs

```diff
@@ -1,6 +1,8 @@
 using System;
+using System.Collections;
 using System.Collections.Generic;
 using System.Collections.Specialized;
+using System.Linq;
```

Comment:
We try to avoid using Linq extension methods, most of them are generating too many unrequited allocations